doc/api/child_process.md

```diff
@@ -177,6 +177,9 @@ directly by the shell and special characters (vary based on
 [shell](https://en.wikipedia.org/wiki/List_of_command-line_interpreters))
 need to be dealt with accordingly:
 
+If the `shell` option is passed as `false`, then certain shell behaviours such
```

Comment:
line 174 above isn't correct anymore, its only sometimes true that a shell is  spawned.